dapper/mods/VL20/__init__.py

```diff
@@ -1,10 +1,12 @@
 """
-One scale modified model of Lorenz96 which added one thermodynamic variable.
+Single-scale Lorenz-96 with an added thermodynamic component.
+
+Refs: `bib.vissio2020mechanics`
 """
 import numpy as np
 
 
-class model():
+class model_instance():
```

Comment:
This is probably not the best class name. I don't know why I didn't follow convention. I suggest renaming to ModelConfig

This should also be done to QG and LUV models, but maybe in another PR

Reply:
I agree ModelConfig seems better but I think this is something we should take a note for the next PR related to parameter estimation and documentation migration, which require some thoughts on the changes of mods.